.env.development

```
@@ -1,18 +1,14 @@
 KYC_URL="https://api.digitalid-sandbox.com/products/kyc/1/"
 KYC_CLIENT_ID="<KYC_CLIENT_ID>"
-KYC_CLIENT_SECRET="<KYC_CLIENT_SECRET>"
 GPIB_API_ENDPOINT="https://testapi.getpaidinbitcoin.com.au"
-HTTP_BASIC_USER="HTTP_BASIC_USER"
-HTTP_BASIC_PASS="HTTP_BASIC_PASS"
 COINSTASH_SIGNUP_ENDPOINT="https://coinstash.com.au/api/auth/signup"
-COINSTASH_TOKEN="<COINSTASH_TOKEN>"
 EC_SIGNUP_ENDPOINT="https://www.googleapis.com/identitytoolkit/v3/relyingparty/signupNewUser?key=<EC_KEY>"
 EC_VERIFY_ENDPOINT="https://api.easycrypto.com.au/apiv2/verify.php"
 APP_DEEPLINK_URL="idem://exchange/vendors"
-WALLET_PRIVATE_KEY="<WALLET_PRIVATE_KEY>"
 WALLET_ADDRESS="0x90494cA0a639Bf52E65511afF6458b9AB5eA77De"
 IDEM_URL="https://idem.com.au/"
-OTP_HASHING_SECRET="SUPER_SECRET_KEY_FOR_HASHING"
-MESSAGEBIRD_API_KEY="MESSAGEBIRD_API_KEY"
 DIGITALSURGE_SIGNUP_ENDPOINT="https://sita.digitalsurge.io/api/partner/createcustomer/"
-DIGITALSURGE_PARTNER_TOKEN="<DIGITALSURGE_PARTNER_TOKEN>"
\ No newline at end of file
+FROM_EMAIL_ADDRESS="support@dltx.io"
+PROXY_API_URL= "https://uat-proxy.idem.com.au/user"
+WEBSITE_URL="https://www.idem.com.au/"
+JWT_EXPIRATION_SECONDS="21600"
```

do we need quotes here?
```suggestion
JWT_EXPIRATION_SECONDS=21600
```